packages/styles/scss/components/masthead/_masthead.scss

```diff
@@ -308,13 +316,30 @@ $search-transition-timing: 95ms;
     }
   }
 
+  .sub-content-left {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 1px;
+    height: 100%;
+  }
+
+  .sub-content-right {
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 1px;
+    height: 100%;
+  }
```

Comment:
Do we want to add `.bx--` prefix? If not, probably `.dds--` prefix?

Reply:
yes agree! Good catch!

Reply:
just updated! Thanks!